pkg/collector/adapters/config_validate.go

```diff
@@ -75,19 +68,19 @@ func GetEnabledReceivers(logger logr.Logger, config map[interface{}]interface{})
 			//Safe Cast
 			pipelineV, ok := pipelineID.(string)
 			if !ok {
-				return nil, fmt.Errorf("PipelineID is not a string: %v", pipelineV)
+				return nil
 			}
 			//Condition will get information if there are multiple configured pipelines.
 			if len(pipelineV) > 0 {
 				pipelineDesc, ok := pipelineCfg.(map[interface{}]interface{})
 				if !ok {
-					return nil, fmt.Errorf("pipeline was not properly configured")
+					return nil
 				}
 				for pipSpecID, pipSpecCfg := range pipelineDesc {
 					if pipSpecID.(string) == "receivers" {
 						receiversList, ok := pipSpecCfg.([]interface{})
 						if !ok {
-							return nil, fmt.Errorf("no receivers on pipeline configuration %q", receiversList...)
+							return nil
```

Comment:
Is this an error, though? Perhaps we should just continue with the next pipeline?

Reply:
The same comment applies to other places within this `for` loop.